### PR TITLE
[EAGLE-4050]-consume-token

### PR DESCRIPTION
**Pull request by phatvo9**
### What
Add session-token as alternative authentication
### Why
To be able to use SDK in other hosted modules like `module-eval` as we pass to it a token instead of PAT
### How
add `token` arg in `__init__` method

### CODE REVIEW

**Review by sanjaychelliah — COMMENTED** [review]
Great addition @phatvo9. Added some comments...

File: clarifai/client/app.py [hunk 1]
```diff
@@ -585,17 +570,18 @@ def module(self, module_id: str, module_version_id: str = "", **kwargs) -> Modul
...
   def inputs(self,):
     """Returns an Input object.
 
     Returns:
         Inputs: An input object.
     """
-    return Inputs(self.user_id, self.id, base_url=self.base, pat=self.pat)
+    return Inputs.from_auth_helper(
+        self.auth_helper
+    )  #self.user_id, self.id, base_url=self.base, pat=self.pat, token=self.token)
```
Comment (ackizilkale) [inline clarifai/client/app.py:584]: cleanup here

File: clarifai/client/app.py [hunk 2]
```diff
@@ -532,9 +518,9 @@ def model(self, model_id: str, model_version_id: str = "", **kwargs) -> Model:
     kwargs = self.process_response_keys(dict_response['model'], 'model')
     kwargs[
         'model_version'] = response.model.model_version if response.model.model_version else None
-    kwargs.update({'base_url': self.base, 'pat': self.pat})
+    #kwargs.update({'base_url': self.base, 'pat': self.pat, 'token': self.token,})
```
[the remark this answers is not in the document]
Comment (phatvo9) [reply clarifai/client/app.py]: removed

**Review by sanjaychelliah — APPROVED** [review]
LGTM!!

**Pull request merged**